src/main/java/org/mitre/synthea/world/concepts/Claim.java

```diff
@@ -157,10 +157,20 @@ private void assignCosts(PlanRecord planRecord) {
       }
 
       plan.incrementCoveredEntries(this.entry);
+
+      if (planRecord.getOutOfPocketExpenses().compareTo(plan.getMaxOop()) > 0) {
+        // TODO - This will only trigger after a person has already paid more than their Max OOP.
```

Comment:
I left this in this state since this assign costs logic isn't very conducive to adding Max OOP. The way the conditionals check for who should pay at every step and factoring in the deductible would make a full Max OOP convoluted.
Let me know if I should add this full logic in anyway, remove Max OOP entirely, or leave it as is. I think Max OOP makes sense as a feature, especially in regards to Medicare being a coinsurance plan.

Reply:
I'm torn on this one. OTOH, I can see the attraction of keeping it simple but OTOH it wouldn't be great to have everyone regularly paying over the Max OOP by varying amounts. I'd say fix it if possible.